module/userProjects/helper.js

```diff
@@ -1291,9 +1292,11 @@ module.exports = class UserProjectsHelper {
                     userId: userId,
                     isAPrivateProgram: targetedSolutionId.data.isATargetedSolution ? false : true
                 }, ["_id"]);
-                if( projectDetails.length > 0 ) {
+                if( projectDetails.length < 0 ) {
```

Comment:
why is this > changed to < ?? @praveenKDass

Reply:
i was testing will revert it now